index.js

```diff
@@ -0,0 +1,142 @@
+const buttonElement = document.querySelector('button.add-form-button');
+// console.log(document.querySelector('button.add-form-button'))
+const listElement = document.querySelector('.comments');
+// console.log(document.querySelector('.comments'));
+const inputNameElement = document.querySelector('.add-form-name');
+// console.log(document.querySelector('.add-form-name'));
+const textareaElement = document.querySelector('.add-form-text');
+// console.log(document.querySelector('.add-form-text'));
+// console.log(listElement.innerHTML);
+
+
+const myDate = new Date();
+const options = {
+  year: "2-digit",
+  month: "2-digit",
+  day: "2-digit",
+  timezone: "UTC",
+  hour: "numeric",
+  minute: "2-digit"
+};
+
+options.hour = "2-digit";
+const date = myDate.toLocaleDateString("ru-Ru", options).split(', ').join(' ');
+
+// buttonElement.disabled = true;
+// inputNameElement.oninput = () => {
+// if(inputNameElement.value === ''){
+//     buttonElement.disabled = true;
+// } else {
+//     buttonElement.disabled = false;
+// }
+// };
+
+// textareaElement.oninput = () => {
+//     textareaElement.value === '' ? buttonElement.disabled = true : buttonElement.disabled = false;
+// };
+
+
+const comments = [
+  {
+    name: 'Глеб Фокин',
+    date: '12.02.22 12:18',
+    comment: 'Это будет первый комментарий на этой странице',
+    likesCounter: 3,
+    isLike: false,
+  },
+  {
+    name: 'Варвара Н.',
+    date: '13.02.22 19:22',
+    comment: 'Мне нравится как оформлена эта страница! ❤',
+    likesCounter: 75,
+    isLike: true,
+  },
+];
+
+const renderComments = () => {
+  const commentHtml = comments.map((comment, index) => {
+    return `<li class="comment">
+<div class="comment-header">
+  <div>${comment.name}</div>
+  <div>${comment.date}</div>
+</div>
+<div class="comment-body">
+  <div class="comment-text">
+    ${comment.comment}
+  </div>
+</div>
+<div class="comment-footer">
+  <div class="likes">
+    <span class="likes-counter">${comment.likesCounter}</span>
+    <button class="${comments[index].isLike ? "like-button -active-like" : "like-button"}" data-index='${index}'></button>
```

Comment:
Можно использовать переменную comment для доступа к текущему элемену массива, вместо того чтобы доставать его по индексу:

```suggestion
    <button class="${comment.isLike ? "like-button -active-like" : "like-button"}" data-index='${index}'></button>
```

Reply:
Спасибо, что подсказал! Так выглядит намного лучше)